sdcm/nemesis.py

```diff
@@ -100,6 +101,13 @@
 from test_lib.cql_types import CQLTypeBuilder
 
 LOGGER = logging.getLogger(__name__)
+# NOTE: following lock is needed in the K8S multitenant case
+NEMESIS_LOCK = Lock()
```

Comment:
if it's used only for k8s multitenant, should it be named like that ?

Reply:
Technically, it is used for any backend and any nemesis. Just in common case it doesn't do any effect.